src/views/UserDetail/UserDetail.js

```diff
@@ -559,6 +564,8 @@ class UserDetail extends React.Component {
 
       patronBlocks = orderBy(patronBlocks, ['metadata.createdDate'], ['desc']);
 
+      if (!this._isMounted) return;
```

Comment:
Maybe it is better to add `if (!this._isMounted) return;` at the beginning of function? In this case we will not waste time calculating `patronBlocks`.